int32 channel
bool value
---
